packages/shared/src/utils/benchmark-handler/benchmark-handler.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * The function is intended to be used as a callback for the React Profiler.
+ * @param {string} id
+ * @param {'mount'|'update'|'nested-update'} phase
+ * @param {number} actual_time
+ * @param {number} base_duration
+ * @param {number} start_time
+ * @param {number} commit_time
+ */
+export const benchmarkHandler = (id, phase, actual_time, base_duration, start_time, commit_time) => {
+    if (phase === 'mount') {
```

Comment:
dont we need other phases to be implemented?

Reply:
For now, our concern was to get measure the initial load, but for that we can make a `HOC` which will wrap the profiler component and on each place we can pass some other function to additionally measure the `update` phase if needed.